assets/src/edit-story/components/canvas/pagemenu/index.js

```diff
@@ -222,6 +242,35 @@ function PageMenu() {
               {isRTL ? <LeftArrow /> : <RightArrow />}
             </Icon>
           </WithTooltip>
+          <Space />
+          {enableAnimation &&
```

Comment:
Should "animate" button still be shown even if there are no animations at all on the page?

Reply:
This is what I had initially, then I thought it was nicer if it was disabled if there were no animations to play. happy to do either. I couldn't find anywhere in the spec where it specified the expected behavior here, but if you've seen this around can certainly update real quick.

Reply:
Disabled is good.